spec/factories/topic.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :topic do
+    name { Faker::Company.industry }
```

Comment:
I'll suggest to use a sequence here to avoid the case where `Faker::Company.industry` would return a not unique value. [See more here](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#sequences)

Reply:
Right, I think he used it as we talked about this 🤔 , i don't know why it's not here